config/initializers/limits.rb

```diff
@@ -4,3 +4,8 @@
 def secrets_version_limit
   ( ENV['SECRETS_VERSION_LIMIT'] || 20 ).to_i
 end
+
+def policies_version_limit
+  ( ENV['POLICIES_VERSION_LIMIT'] || 20 ).to_i
```

Comment:
20 need to be as variable otherwise we will need to change it every time.

Reply:
i think it can stay as 20 as this is the place where we set the variable